src/lab-tabs/actions/send-back-patient-action.component.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import styles from "./actions.scss";
+import { showModal } from "@openmrs/esm-framework";
+import { Button } from "@carbon/react";
+
+interface SendBackPatientActionProps {
+  patientUuid: string;
+}
+
+const SendBackPatientAction: React.FC<SendBackPatientActionProps> = ({
```

Comment:
This logic for queues should be done on the service queues module.

Reply:
Resolved